module3/projects/take_home/presentation_guidelines.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Take-Home Challenge Presentation Guidelines
+layout: page
+---
+
+## Presentation Guidelines
+
+This guide will serve as a general outline for how students can present their code during a take-home challenge presentation. 
+
+<section class="note">
+### Note
+
+Keep in mind that some companies will have their own guidelines on presenting which takes precenedence over this guideline.
+</section>
+
+**Prior to the presentation:**
+
+- Clear your screen of all windows or tabs except the ones relevant to your presentation. 
+
+- Have any servers fired up and ready to go.
+
+<!-- Note: Do we want them to demo in postman if there is a frontend? -->
+- Have Postman setup in advance to show your endpoints (BE).
+
+
+## Suggested steps for presenting your project:
+### 1. Overview
+  - Start with an overview of the app that you built, summarizing the tasks of the challenge. This can be as simple as verbally stating the main functions of the app.
+
+### 2. Functionality
+
+- **Back End**
+  - Start with Postman and show how your app works through all your endpoints.
+  - If you have a testing suite, now would be a great time to run it and show your test coverage.
+
+- **Front End**
+  - Run through the app, clicking on all the links, entering text into the fields, show the user experience of the application, any responsive design, etc.
```

Comment:
They will have learned Cypress testing, so we could add the same line `If you have a testing suite, now would be a great time to run it and show your test coverage.` here.